fairseq/trainer.py

```diff
@@ -433,8 +439,11 @@ def maybe_no_sync():
             self.meters['wpb'].update(ntokens)
             self.meters['bsz'].update(nsentences)
             self.meters['gnorm'].update(grad_norm)
+            # the comparison below introduces too many .item() calls and slows
+            # down tpu
             self.meters['clip'].update(
-                1. if grad_norm > self.args.clip_norm and self.args.clip_norm > 0 else 0.
+                0.
+                #1. if grad_norm > self.args.clip_norm and self.args.clip_norm > 0 else 0.
```

Comment:
We don't need this comment right and all other comments in the PR?

Reply:
yeah, this line causes a _local_scalar_dense, so I replaced it with 0. The meter is essentially invalid. I'll cancel this meter if xla.

Reply:
actually, may I handle this in a separate pr? I have one in the works that will change the logic around meters a bit.